src/domain/accounts/services/src/services/predefined_accounts_registrator.rs

```diff
@@ -56,9 +56,13 @@ impl PredefinedAccountsRegistrator {
         account_config: &AccountConfig,
     ) -> Result<(), InternalError> {
         // TODO: Revisit if batch property setting will be implemented
-        for (name, value) in [AccountPropertyName::is_admin(account_config.is_admin)] {
+        let Some(account_properties) = account_config.properties.as_ref() else {
+            return Ok(());
+        };
+
+        for name in account_properties {
             self.rebac_service
-                .set_account_property(account_id, name, &value)
+                .set_account_property(account_id, *name, &boolean_property_value(true))
                 .await
                 .int_err()?;
         }
```

Comment:
I think this solution may be fragile, and it is better to explicitly set the values. We have default values in case something is not explicitly specified